src/mSupplyMobileApp.js

```diff
@@ -211,23 +197,11 @@ class MSupplyMobileAppContainer extends React.Component {
     );
   };
 
-  renderSyncState = () => {
-    const { syncState } = this.props;
-    return (
-      <TouchableOpacity
-        style={{ flexDirection: 'row' }}
-        onPress={() => this.setState({ syncModalIsOpen: true })}
-      >
-        <SyncState state={syncState} />
-      </TouchableOpacity>
-    );
```

Comment:
Not needed, all handled internally in the `<SyncState/>` component - no need to hold this state in this root component